### DISCUSSION TITLE
New version of Openagbfirm?

### DISCUSSION THREAD

**Discussion by Uretrozone** [profi200/open_agb_firm#47] [General]
when is a new version coming out

**Comment by profi200** [comment]
When i find the time and motivation to do it. I will see if i can make one until the weekend.

**Reply by Uretrozone** [reply]
Nice what features is the new version gonna have? also am i gonna be able to turn my brightness to a 100?

**Reply by profi200** [reply]
It's mainly these 2 pull requests. Everything else are not user facing changes:

#35 
#40

**Comment by profi200** [comment]
https://github.com/profi200/open_agb_firm/releases/tag/alpha_2021-11-21
